src/ext/resources/legacygraphqlapi/schema.graphqls

```diff
@@ -2203,6 +2335,41 @@ type Plan {
     debugOutput: debugOutput!
 }
 
+type PlanConnection {
+    searchDateTime: PlanDateTime
+
+    locations: PlanPlaces
+
+    routingErrors: [RoutingError!]!
+
+    debugInformation: PlanDebugInformation!
+
+    edges: [PlanEdge]
+
+    pageInfo: PageInfo!
+}
+
+type PlanDateTime {
+    date: DateTime
+    timeZone: TimeZone
```

Comment:
If we want to add the timezone, we can also think about using Java `ZonedDateTime`'s string format: `2023-06-15T12:43:55+03:00[Europe/Helsinki]`

Reply:
I think we can remove this if we have the offset datetime.

Reply:
https://github.com/opentripplanner/OpenTripPlanner/pull/5185/commits/910717a739f12e99b3a445c338fd24a8e4b73637